protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -1255,10 +1258,10 @@ func renderServices(services []*descriptor.Service, paths *openapiPathsObject, r
 				if b.Body != nil {
 					// Recursively render fields as definitions as long as they contain path parameters.
 					// Special case for top level body if we don't have a body field.
-					var schema openapiSchemaObject
+					var schema *openapiSchemaObject
```

Comment:
Do we need this declaration? I feel like we can just use
```
schema :=
```
on line 1264?

Reply:
true, removed